clients/wallet/src/horizon/mod.rs

```diff
@@ -8,6 +8,3 @@ mod tests;
 
 pub use horizon::*;
 pub use traits::HorizonClient;
-
-// todo: change to Slot
-pub type Ledger = u32;
```

Comment:
`Slot` is available and can be used: 
   https://github.com/pendulum-chain/spacewalk/blob/bda384bcc6db3da88198c714851a261dcdea1063/clients/wallet/src/types.rs#L6

so there's really no need for `type Ledger`.